iotwifi/wpacfg.go

```diff
@@ -93,6 +93,8 @@ func (wpa *WpaCfg) StartAP() {
 ssid=` + wpa.WpaCfg.HostApdCfg.Ssid + `
 hw_mode=g
 channel=` + wpa.WpaCfg.HostApdCfg.Channel + `
+ctrl_interface=/var/run/hostapd
+ctrl_interface_group=0
```

Comment:
This enabled hostapd_cli

Reply:
Thoughts on adding `WpaCfg.CtrlInterface` and `WpaCfg.CtrlInterfaceGroup` to the config instead of hard coding these values? Should be ok to hardcode them for now for our purposes but if we want to submit this patch upstream it would be good to make it configurable for other devices / distros / configuration

Reply:
I don't think it makes sense as a user configuration since core logic in the service relies on it.